src/store/ducks/opensrp/events/index.tsx

```diff
@@ -0,0 +1,106 @@
+/** Events redux module */
+import { Dictionary, get, keyBy } from 'lodash';
+import { Store } from 'redux';
+import { AnyAction } from 'redux';
+import SeamlessImmutable from 'seamless-immutable';
+import { Event, extractEvents, RawEvent } from './utils';
```

Comment:
@p-netm @moshthepitt - where do you think is the best place for these type definitions?

Reply:
I can create an Issue to discuss and implement.

Reply:
#736